runtime/frequency/src/lib.rs

```diff
@@ -945,6 +945,7 @@ impl pallet_handles::Config for Runtime {
 impl pallet_sudo::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type RuntimeCall = RuntimeCall;
+	type WeightInfo = weights::pallet_sudo::SubstrateWeight<Runtime>;
```

Comment:
Now that I think of it, if we are just using substrate weights, we could map this directly to the weights from the pallet like we used to before moving to our own weights.

Reply:
Makes sense to use original weights from pallet